### DISCUSSION TITLE
Failed to deploy with no information why it failed verbose doesn't help

### DISCUSSION THREAD

**Discussion by JefStat** [serverless/serverless#11190] [Q&A] [answered]
I'm running 
`sls deploy --package .serverless --stage prod --verbose`
To test credentials on a new CI deployment

The deploy fails but even with verbose and/or SLS_DEBUG=* the same output is show not telling me which iam permission failed as I expect them too. 

```
Running "serverless" from node_modules

Deploying x to stage prod (us-west-2)

Retrieving CloudFormation stack

✖ Stack x-prod failed to deploy (1s)
Environment: darwin, node 16.15.1, framework 3.19.0 (local) 3.19.0v (global), plugin 6.2.2, SDK 4.3.2
Credentials: Local, "default" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Access Denied
```

**Comment by pgrzesik** [comment, marked as answer]
Hey @JefStat, thanks a lot for sharing, we've had a few examples of this reported but I was not able to reproduce it as of yet. Did you try to deploy it with very broad permissions and it worked, however, it started failing when you trimmed down the permissions?

**Reply by JefStat** [reply]
I've started from a few permissions and have been broadening them. I'm trying to step through the node_modules now to determine the point it fails at.

**Reply by JefStat** [reply]
Found an error Looks like it's an `AWS_S3_LIST_OBJECTS_V2_ACCESS_DENIED`
![image](https://user-images.githubusercontent.com/583018/176712521-d94230c5-c688-48e8-b482-e11b8f8a958c.png)

**Reply by JefStat** [reply]
The following iam error is descriptive as I expected after fixing my ListBucket permission (simple wrong bucket name mistake). eg:
```
Error:
User: arn:aws:iam::     is not authorized to perform: cloudformation:DeleteChangeSet on resource: arn:aws:cloudformation  because no identity-based policy allows the cloudformation:DeleteChangeSet action
```

**Reply by pgrzesik** [reply]
Thanks a lot for your investigation and providing detailed description of your findings, I will look into it 👍

**Reply by pgrzesik** [reply]
Thanks @JefStat - I've proposed a PR that addresses these errors in a few places https://github.com/serverless/serverless/pull/11217

**Reply by SushilMallRC** [reply]
@JefStat JefStatI was facing same problem , I just remove profile from serverless.yml and it got resolved.
